### PR TITLE
Restoration of condensed Intro to Web Development for WebDev101

### PR DESCRIPTION
**Pull request by JonathanYiv**
Features:
1) Nearly similar wording, either copy/pasting previous lesson text or slightly modifying it to improve the flow. Including Erik's "Conclusion" section which I could tell he probably was proud of.
2) 60% reduction of content for brevity.
3) Added "Why Odin?" and "Motivation" section because I felt they were helpful.
4) Line at beginning indicating this lesson is skippable.

### CODE REVIEW

File: web_development_101/introduction_to_web_development.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+If you know why you are here and that you want to learn web development, skip this lesson.
```
Comment (KevinMulhern) [inline web_development_101/introduction_to_web_development.md]: I think we can remove this line, this lesson is short so It's not a big commitment to go through it and theres a lot of stuff here that everyone should read even if they know what they are after 😄

**Comment by KevinMulhern** [comment]
Great work @JonathanYiv 💯

**Comment by JonathanYiv** [comment]
Line has been removed. Is there additional work to do to add it to the curriculum?

**Comment by KevinMulhern** [comment]
Yeah it will need to be added to the seeds file here https://github.com/TheOdinProject/theodinproject/blob/master/db/seeds.rb

I think this lesson would fit in perfectly after the "how this course will work" lesson on web dev 101. What do you think?

**Comment by JonathanYiv** [comment]
Sounds good to me! I'll make a PR for the seeds file.

**Pull request merged**